### PR TITLE
IAS-11362: Update dependencies and some exclusions

### PR DESCRIPTION
**Pull request by emcanearney-r7**
### Description
Updating to a newer version for bamboo-web
Excluding some unneeded dependencies which had known snyk issues
Tidying up versions to all be within the properties block

### Motivation and Context
Clean up and update of code.

### How Has This Been Tested?
atlas-clean & atlas-run locally, testing on staging server.

### Types of changes
All pom.xml changes, some updates to existing deps, some exclusions for unneeded ones.

### Checklist:
- [x] I have updated the documentation accordingly (if changes are required).
- [x] I have tested the changes locally.

### CODE REVIEW

**Review by nward-r7 — COMMENTED** [review]
@emcanearney-r7 I notice some other versions have been increased also. Is that for snyk purposes?

**Comment by emcanearney-r7** [comment]
@nward-r7 , yes, some had newer versions that had less known vulnerabilities so I updated where I could.

**Review by nward-r7 — DISMISSED** [review]

**Review by caallison-r7 — DISMISSED** [review]

**Review by rcraig-r7 — DISMISSED** [review]

**Review by scarr-r7 — DISMISSED** [review]

**Review by nward-r7 — DISMISSED** [review]

**Review by rcraig-r7 — DISMISSED** [review]

File: manifest.json [hunk 1]
```diff
@@ -23,6 +23,11 @@
     },
     "version": "1.2.2",
```
Comment (mmacrory-r7) [inline manifest.json]: I think you might need to update the version here to 1.2.3

Comment (mmacrory-r7) [reply manifest.json]: You will also need version bumps in help.md and InsightAppSecHelper.java

**Review by mmacrory-r7 — DISMISSED** [review]

**Review by scarr-r7 — DISMISSED** [review]

**Review by caallison-r7 — DISMISSED** [review]

**Review by caallison-r7 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @zyoutz-r7 @hwilsey-r7 @tschmidtke-r7
\ No newline at end of file
+* @jgamble-r7 @emcanearney-r7 @nward-r7 @CameronAllison97-r7 @mmacrory-r7 @rcraig-r7 @clavery-r7
```
Comment (mmacrory-r7) [inline .github/CODEOWNERS:1]: is this the right name for cameron?

**Review by jgamble-r7 — APPROVED** [review]

**Review by mmacrory-r7 — APPROVED** [review]

**Pull request merged**